test/tests/problems/eigen_problem/tests

```diff
@@ -58,25 +67,46 @@
     cli_args = 'Executioner/solve_type=MF_MONOLITH_NEWTON Outputs/file_base=monolith_newton'
     csvdiff = 'monolith_newton_eigenvalues_0001.csv'
     slepc_version = '>=3.8.0'
+    requirement = "Eigenvalue system should be able to solve a nonlinear eigenvalue problem using Newton"
   [../]
 
   [./ne_deficient]
     type = 'CSVDiff'
     input = 'ne_deficient_b.i'
     csvdiff = 'ne_deficient_b_eigenvalues_0001.csv'
     slepc_version = '>=3.8.0'
+    requirement = "Eigenvalue system should be able to solve a deficient eigenvalue problem"
   [../]
 
   [./nonlinear_laplace]
     type = 'CSVDiff'
     input = 'ane.i'
     csvdiff = 'ane_eigenvalues_0001.csv'
     slepc_version = '>=3.8.0'
+    requirement = "Eigenvalue system should be able to compute a nonlinear eigenvalue problem"
   [../]
+
   [./coupled_system]
     type = 'CSVDiff'
     input = 'ne_coupled.i'
     csvdiff = 'ne_coupled_eigenvalues_0001.csv'
     slepc_version = '>=3.8.0'
+    requirement = "Eigenvalue system should be able to compute a coupled nonlinear eigenvalue problem"
+  [../]
+
+  [./eigen_scalar_kernel]
+    type = 'CSVDiff'
+    input = 'scalar.i'
+    csvdiff = 'scalar_out_eigenvalues_0001.csv'
+    slepc_version = '>=3.8.0'
+    requirement = "Eigenvalue system should be able to handle scalar kernels"
   [../]
+
+  [./dg_krylovschur]
+    type = 'CSVDiff'
+    input = 'dg_krylovschur.i'
+    csvdiff = 'dg_krylovschur_out_eigenvalues_0001.csv'
+    slepc = true
```

Comment:
Could you add `slepc_version = '>=3.8.0'`

Reply:
I was going to added this, but I didn't see Krylov-Schur blocked by a version requirement in `SlepcSupport`. Can do

Reply:
Yes, you are right. In theory, it should work with something older than 3.8.0. But I could not remember  if I did any tests again the older PETSc. It was why I was asking we simply skip the older PETSc so that we do not need to go back and check if it works.